lib/components/app/app-menu.tsx

```diff
@@ -294,39 +251,21 @@ export default injectIntl(
 /**
  * Renders a label and icon either from url or font awesome type
  */
-const IconAndLabel = ({
+const Icon = ({
   iconType,
-  iconUrl,
-  label
+  iconUrl
 }: {
   iconType: string
   iconUrl: string
-  label: string
 }): JSX.Element => {
-  const intl = useIntl()
   // FIXME: add types to context
   // @ts-expect-error No type on ComponentContext
   const { SvgIcon } = useContext(ComponentContext)
-
-  return (
-    <span>
-      {/* TODO: clean up double ternary */}
-      {iconUrl ? (
-        <img
-          alt={intl.formatMessage(
-            {
-              id: 'components.AppMenu.menuItemIconAlt'
-            },
-            { label }
-          )}
-          src={iconUrl}
-        />
-      ) : iconType ? (
-        <SvgIcon iconName={iconType} />
-      ) : (
-        <ExternalLinkSquareAlt />
-      )}
-      {label}
-    </span>
+  return iconUrl ? (
+    <img alt="" src={iconUrl} />
```

Comment:
Are you sure about the blank alt? I get how the UX might be better but if we make it invisible to screen readers shouldn't it have a `role="presentation"`?

Reply:
Benefits of `alt=""` are explained here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-alt

Reply:
`alt=""` is used by screen readers and also by visual browsers (e.g. when the `src` is broken.)

Reply:
Ok I trust you on this one